tests/test_middleware_interface.py

```diff
@@ -109,24 +123,28 @@ def test_init(self):
     def test_prep_butler(self):
         """Test that the butler has all necessary data for the next visit.
         """
-        with self.assertLogs(self.logger_name, level="INFO") as cm:
-            self.interface.prep_butler(self.next_visit)
-        msg = f"INFO:{self.logger_name}:Preparing Butler for visit '{self.next_visit}'"
-        self.assertEqual(cm.output, [msg])
-        # TODO: Test that we have appropriate refcats?
+        self.interface.prep_butler(self.next_visit)
+        self.assertEqual(self.butler.get('camera',
+                                         instrument=instname,
+                                         collections=[f"{instname}/calib/unbounded"]).getName(), instname)
+        # TODO: check that we got the correct refcat shards. How to do that?
```

Comment:
For the short term, you could precalculate and hard-code for your known test data.

Reply:
Part of the question was "what *are* the correct refcat shards?" See DM-34112 for further investigation.

At least I have some kind of test in here now.